docs/rule/require-lang-attribute.md

```diff
@@ -0,0 +1,52 @@
+# require-lang-attribute
+
+A missing `lang` attribute can cause an application to fail legal conformance for digital accessibility requirements.
+
+This rule's objective is to ensure that Ember applications achieve [WCAG Success Criterion 3.1.1: Language of Page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html). The state of the `lang` attribute has a usability impact on the experience of users that require screen-reading assistive technology. When the attribute is properly assigned:
+
+> "Both assistive technologies and conventional user agents can render text more accurately when the language of the Web page is identified. Screen readers can load the correct pronunciation rules. Visual browsers can display characters and scripts correctly. Media players can show captions correctly. **As a result, users with disabilities will be better able to understand the content.**"
+>
+> **Source: [WCAG Success Criterion 3.1.1: Intent](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#intent)**
+
+When the language of the page cannot be identified, the integrity of the above information cannot be guaranteed.
+
+Consider the following use case:
```

Comment:
Are these bullets listing multiple use cases or a single use case? Basically, should I be reading this list as if AND'ed or OR'ed?